string-manipulation-1/src/capitalize.js

```diff
@@ -21,12 +21,12 @@
 function capitalize(word) {
   var output = '';
   var letter = '';
-  for (var i = 0; i < word.length; i++) {
-    if (i === 0) {
-      letter = word[i].toUpperCase();
-      output += letter;
-      letter = '';
-    } else {
+  var i = 0;
+  letter = word[i].toUpperCase();
+  output += letter;
+  letter = '';
+  for (i = 1; i < word.length; i++) {
+    if (i !== 0) {
```

Comment:
If you are starting at `1`, why do we need to check if the index is not `0`?

Reply:
Aahhhh! Good point haha